Octokit/Models/Response/UserType.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Octokit
+{
+    public enum UserType
```

Comment:
I don't understand. Are you saying we can't use `AccountType` here because the API endpoint that needs `UserType` returns "Organization" and the API endpoint that needs `AccountType` returns "Org"? Ugh.